.github/issue_template.md

```diff
@@ -0,0 +1,23 @@
+### Subject of the issue
+
+Describe your issue here.  Remember, this is _not_ a place to ask questions.  For that, go to [spectrum](https://spectrum.chat/mdx)
+
+### Your environment
+
+*   Name and version of operating system
+*   Names and version of required packages
+*   Version of node and npm, or names and versions of browser
```

Comment:
True, it isn’t always useful, but sometimes it can speed things up to immediately see “oh, this may be a windows issue”, and skip one back and forth if that already confirmed.
Anyway, I’m fine removing it! WDYT?

Reply:
Actually yes, I agree.